package.json

```diff
@@ -22,13 +22,13 @@
   "main": "mqtt.js",
   "types": "types/index.d.ts",
   "scripts": {
-    "test": "node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly --",
+    "test": "node_modules/.bin/nyc --reporter=lcov --reporter=text-summary ./node_modules/mocha/bin/_mocha",
     "pretest": "standard | snazzy",
     "tslint": "if [[ \"`node -v`\" != \"v4.3.2\" ]]; then tslint types/**/*.d.ts; fi",
     "typescript-compile-test": "tsc -p test/typescript/tsconfig.json",
     "typescript-compile-execute": "node test/typescript/*.js",
     "typescript-test": "npm run typescript-compile-test && npm run typescript-compile-execute",
-    "browser-build": "rimraf dist/ && mkdirp dist/ && browserify mqtt.js -s mqtt > dist/mqtt.js && uglifyjs < dist/mqtt.js > dist/mqtt.min.js",
+    "browser-build": "rimraf dist/ && mkdirp dist/ && webpack",
```

Comment:
Is this change needed for this PR? Maybe we should focus on the browser on the next PR.

Reply:
I can remove this from the PR, but it is broken so the Github Actions will fail.